2-mandatory/7-lane-names.js

```diff
@@ -6,7 +6,9 @@
   HINT: string and array methods that could be helpful (indexOf, filter)
 */
 
-function getLanes() {}
+function getLanes(arr) {
```

Comment:
I think we all need to work on how to name our variables to make our code easier to read